test/pythonml/test_workflow_integration.py

```diff
@@ -143,7 +143,12 @@ def simulate_workflow(self, in_test: List[str]):
 
             sys.stdout.write(stdout.decode())
 
-            self.assertFalse(stderr, f"\nSTDERR:\n{stderr.decode()}")
+            # Sometimes, Matplotlib will write to stderr to state that it's building the font cache.
```

Comment:
Should we add https://github.com/ocropus/ocropy/issues/204 in the comment?

Reply:
Added to comment